### PR TITLE
[11.0][MIG] partner_contact_role

### PR DESCRIPTION
**Pull request by simahawk**
Fwd port for #521

### CODE REVIEW

**Comment by yajo** [comment]
Let's set as wip until #521 is merged, to avoid duplicating reviews.

**Review by mpanarin — APPROVED** [review]
other than small things(copyright year) LGTM

**Comment by grindtildeath** [comment]
@simahawk Can you update #521 and rebase this afterwards ?

**Comment by simahawk** [comment]
I'll wait for #521 to get merged.

**Review by grindtildeath — APPROVED** [review]

**Comment by GSLabIt** [comment]
@simahawk is this ready to merge?

**Comment by simahawk** [comment]
AFAIR yes

**Review by lmignon — APPROVED** [review]

File: partner_contact_role/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    'depends': [
+        'contacts',
+    ],
+    'data': [
+        'security/ir.model.access.csv',
+        'views/res_partner_role.xml',
+        'views/res_partner.xml',
+    ],
+    'demo': [
+    ],
```
Comment (MeetKD) [inline partner_contact_role/__manifest__.py]: It would be great if you can remove empty keys :)

**Pull request merged**